### PR TITLE
Update option philosophy: option requests aren’t accepted anymore

### PR DESCRIPTION
**Pull request by thorn0**
**✨[Preview](https://deploy-preview-8540--prettier.netlify.app/docs/en/next/option-philosophy.html)✨**

### CODE REVIEW

File: docs/option-philosophy.md [hunk 1]
```diff
@@ -3,45 +3,36 @@ id: option-philosophy
...
-- `--prose-wrap` is important to support all quirky markdown renderers in the wild.
+- `--prose-wrap` is important to support all quirky Markdown renderers in the wild.
 - `--html-whitespace-sensitivity` is needed due to the unfortunate whitespace rules of HTML.
 - `--end-of-line` makes it easier for teams to keep CRLFs out of their git repositories.
 - `--quote-props` is important for advanced usage of the Google Closure Compiler.
 
-But others are harder to motivate in hindsight, and usually end up with bike shedding. `--arrow-parens`,
-`--jsx-single-quote`, `--jsx-bracket-same-line` and `--no-bracket-spacing` are not the type of options we want more of. They exist (and are difficult to remove now), but should not motivate adding more options like them.
+But other options are harder to motivate in hindsight: `--arrow-parens`, `--jsx-single-quote`, `--jsx-bracket-same-line` and `--no-bracket-spacing` are not the type of options we’re happy to have. They cause a lot of bike-shedding in teams, and we’re sorry for that. Difficult to remove now, these options exist as a historical artifact and should not motivate adding more options (“If _those_ options exist, why can’t this one?”).
```
Comment (azz) [inline docs/option-philosophy.md:32]:
```suggestion
But other options are harder to motivate in hindsight: `--arrow-parens`, `--jsx-single-quote`, `--jsx-bracket-same-line` and `--no-bracket-spacing` are not the type of options we’re happy to have. They cause a lot of [bike-shedding](https://en.wikipedia.org/wiki/Law_of_triviality) in teams, and we’re sorry for that. Difficult to remove now, these options exist as a historical artifact and should not motivate adding more options (“If _those_ options exist, why can’t this one?”).
```

File: docs/option-philosophy.md [hunk 2]
```diff
@@ -3,45 +3,36 @@ id: option-philosophy
...
+But other options are harder to motivate in hindsight: `--arrow-parens`, `--jsx-single-quote`, `--jsx-bracket-same-line` and `--no-bracket-spacing` are not the type of options we’re happy to have. They cause a lot of [bike-shedding](https://en.wikipedia.org/wiki/Law_of_triviality) in teams, and we’re sorry for that. Difficult to remove now, these options exist as a historical artifact and should not motivate adding more options (“If _those_ options exist, why can’t this one?”).
+
+For a long time, we left option requests open in order to collect feedback and let discussions play out. What we’ve learned during those years is that it’s really hard to measure demand. Prettier has grown a lot in usage. What was “great demand” back in the day is not as much today. GitHub reactions and Twitter polls became unrepresentative. What about all silent users? It looked easy to add “just one more” option. But where should we have stopped? When is one too many? Even after adding “that one final option”, there would always be a “top issue” in the issue tracker.
```
Comment (boyenn) [inline docs/option-philosophy.md:34]: > For a long time, we left option requests open in order to collect feedback and let discussions play out.

would swap that: 

> For a long time, we left option requests open in order to let discussions play out and collect feedback.

( one is the result of the other )

**Comment by alexander-akait** [comment]
@jakeNiemiec Which features you need?

**Comment by thorn0** [comment]
@alexander-akait It was a proposal to mention forks somewhere in the option philosophy. I think that can be added later.

**Pull request merged**